app/projects.html

```diff
@@ -66,7 +66,7 @@ <h3 class="label">Upcoming</h3>
       <h2 class="h2">Platforms</h2>
       <div class="md:col-span-2 body-text flex flex-col gap-50 md:pt-24">
         <p class="flex flex-col gap-20 max-w-[700px]">
-          Projects are the heart of the Library Innovation Lab. We build things. Our projects attempt to make something — big or small — better, simpler, more transparent, or more fun.
+          Tools are the heart of the Library Innovation Lab. We build things. Our platforms each have their own user base and long-term goals.
```

Comment:
I put a hyphen in the third sentence, but I actually think the sentence isn't necessary:
```suggestion
          Tools are the heart of the Library Innovation Lab. We build things.
```